src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -29,6 +32,8 @@ internal sealed partial class VisualStudioProjectTracker : ForegroundThreadAffin
         #region Mutable fields accessed only from foreground thread - don't need locking for access (all accessing methods must have AssertIsForeground).
         private readonly List<WorkspaceHostState> _workspaceHosts;
 
+        private readonly HostWorkspaceServices _workspaceServices;
```

Comment:
The only things I get from it are an `IDeferredProjectProjectWorkspaceService` which I *could* just make into a MEF import, and an `ICommandLineParser` language service.  Any suggestions on getting the `ICommandLineParser` here?